vsintegration/src/FSharp.Editor/KeywordCompletionProvider.fs

```diff
@@ -0,0 +1,133 @@
+﻿// Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.VisualStudio.FSharp.Editor
+
+open System
+open System.Composition
+open System.Threading
+open System.Threading.Tasks
+open System.Runtime.CompilerServices
+
+open Microsoft.CodeAnalysis
+open Microsoft.CodeAnalysis.Completion
+open Microsoft.CodeAnalysis.Options
+open Microsoft.CodeAnalysis.Text
+
+open Microsoft.FSharp.Compiler.SourceCodeServices
+open Microsoft.FSharp.Compiler.SourceCodeServices.ItemDescriptionIcons
+
+module internal Keywords =
+    let completionItems =
+        [
+            "abstract",  """Indicates a method that either has no implementation in the type in which it is declared or that is virtual and has a default implementation."""
```

Comment:
Both FSAC (and XS ?) are using exactly [same list](https://github.com/ionide/FsAutoComplete/blob/master/src/FsAutoComplete.Core/KeywordList.fs) (also added to FSAC by @vasily-kirichenko). So I guess it suggest that we'd like to have some way of reusing it instead of having same code in 3 different places.